autogpt_platform/frontend/src/types/google.ts

```diff
@@ -0,0 +1,27 @@
+type JSONValue =
+  | string
+  | number
+  | boolean
+  | JSONValue[]
+  | {
+      [key: string]: JSONValue;
+    };
+
+export type GTMParams = {
```

Comment:
"GTM" isn't self-explanatory to me, could you expand the acronym or add a comment? Same for "GA" below.

Reply:
No we are keeping the exact same api so it can be copy pasted